dripcoder20/flexmart-auth
# Added Otp Sender #18

Added Otp Sender#18
Added Otp Verification #7
Resend Verification #6
Refactor user registration #4

**To Test**
1. compile vue `npm run dev`
2. serve php `artisan serve --port 22222`
3. visit at [localhost:2222](localhost:2222)
4. click signup
5. enter mobile number then click `verify`
![image](https://user-images.githubusercontent.com/12622991/83346661-2401d600-a351-11ea-83c5-f55e95a70be6.png)

6. type the code above page `*NOTE: tempory to display code`
![image](https://user-images.githubusercontent.com/12622991/83346673-409e0e00-a351-11ea-8331-4cc44f425ec9.png)

7. register by entering details
![image](https://user-images.githubusercontent.com/12622991/83346693-65928100-a351-11ea-9853-1203d9628cd1.png)

8. clicking register should display user account
![image](https://user-images.githubusercontent.com/12622991/83346707-793de780-a351-11ea-88e3-a02c7a05810a.png)
